jenkins/catapult/jenkins/catapult-client-build-base-image-all.groovy

```diff
@@ -43,32 +43,32 @@ pipeline {
 						}
 					}
 				}
-				stage('gcc 11 westmere') {
+				stage('gcc westmere') {
 					steps {
 						script {
 							dispatch_build_base_image_job('gcc-westmere', 'ubuntu', true)
 						}
 					}
 				}
-				stage('gcc 12 [fedora]') {
+				stage('gcc [fedora]') {
 					steps {
 						script {
-							dispatch_build_base_image_job('gcc-12', 'fedora', false)
+							dispatch_build_base_image_job('gcc-latest', 'fedora', false)
 						}
 					}
 				}
 
-				stage('clang 13') {
+				stage('clang prior') {
 					steps {
 						script {
-							dispatch_build_base_image_job('clang-13', 'ubuntu', false)
+							dispatch_build_base_image_job('clang-prior', 'ubuntu', true)
```

Comment:
did you intentionally change false to true?

Reply:
yea, I added a conan build for previous compiler version.